content/posts/blog/2021-01-24_blog_InstantMessagingEsgehtnichtumdieApp

```diff
@@ -0,0 +1,62 @@
+_Dies ist die deutsche Übersetzung des [ursprünglichen Blogeintrags](https://xmpp.org/2021/01/instant-messaging-its-not-about-the-app/)._
+_Vielen Dank für die Korrekturlesung Daniel Brötzmann!_
+
+Kürzlich haben sich mehrere Leute bei mir gemeldet und gefragt, welche Art von Messenger sie jetzt verwenden sollten - sie sagten, dass sie eigentlich nicht verstehen, worauf sie achten sollten und ob sie von einem der allgemein bekannten Messenger zu einem anderen wechseln sollten. 
+Ich fragte mich, wie ich dies beantworten sollte. Natürlich hätte ich einfach für XMPP (Extensible Messaging and Presence Protocol) plädieren können, aber dann dachte ich, dass dies allein vielleicht keine hilfreiche Antwort ist. Oftmals treffen die Leute nur eine schnelle Entscheidung über ihre Kommunikationssoftware, und diese ist in der Regel keine fundierte Wahl; und so werden sie später zu einem anderen Messenger wechseln.
+
+Viele fügen sogar einen weiteren Messenger zu ihrer immer größer werdenden Sammlung von Messengern hinzu, was wahrscheinlich eher frustrierend als hilfreich ist. Das bringt mich zurück zur ursprünglichen Fragestellung: Welches Problem will man hier lösen? Was sind die Absichten? Kann man eine Lösung mit einer besseren technologischen Basis finden, 
+die den Hype vermeidet und bei der man nicht mehrere Messenger-Apps installieren muss?
+
+Verschiedene Menschen haben verschiedene Überlegungen zur Beantwortung dieser Frage - manche benötigen einen effektiven Datenschutz, oder Datenhoheit, oder einfach die Möglichkeit, alle ihre Kontakte von einem Ort aus zu erreichen. Der Wechsel von einem Messaging-System zu einem anderen bedeutet jedoch oft, einige Kontakte zurückzulassen. Viele Messaging-Systeme erfordern außerdem eine Telefonnummer, was für den Datenschutz nicht gerade förderlich ist.
+
+### Souveränität der Kommunikation
+
+Wie Sie vielleicht schon vermutet haben, werde ich für XMPP plädieren. Ich denke, die erste zu treffende Entscheidung sollte nicht sein, welchen Messenger man verwendet, sondern welche zugrundeliegende Software-Technologie. Betrachten Sie zuerst die Wahl der Technologie, bevor Sie von einer Empfehlung zur nächsten springen.
+
+> XMPP ist ein offenes Protokoll, so wie HTTP für das Web ist. Es spielt keine Rolle, wie Ihre Website aussieht, jeder kann mit ihr interagieren. Dieses Prinzip ist die Idee hinter XMPP, allerdings für Instant Messaging.
+
+Bei der XMPP Standards Foundation und bei vielen anderen, die an dieser offenen Technologie beteiligt sind, sind wir der Meinung, dass XMPP als Kommunikationstechnologie eine gute Wahl ist, nicht nur im Hinblick auf den Datenschutz. XMPP gibt es seit über zwanzig Jahren, und in dieser Zeit wurde eine Menge praktischer Erfahrung gesammelt. Dazu gehören viele wichtige Indikatoren für die Wahl der Technologie und die Ermöglichung von Souveränität der Kommunikation; die wichtigen Indikatoren sind:
```

Comment:
[...] gute Wahl ist, und das nicht nur in Hinblick auf Datenschutz.
[...] zwanzig Jahren und in dieser Zeit (Komma)
Dazu gehören viele Kriterien und Indikatoren, die sich als wertvoll für eine fundierte Wahl der Technologie und Ermöglichung von digitaler Souveränität herauskristalisiert haben. Dazu gehönren: